sos/report/plugins/nvme.py

```diff
@@ -26,24 +26,44 @@ class Nvme(Plugin, IndependentPlugin):
     kernel_mods = ('nvme', 'nvme_core')
 
     def setup(self):
+        _subdir1 = "nvme_smartctl"
+        _subdir2 = "nvme_fw_smart_error_logs"
+        _subdir3 = "nvme_list-ns"
+        _subdir4 = "nvme_list-ctrl"
+        _subdir5 = "nvme_id-ctrl_-H"
+        _subdir6 = "nvme_id-ns_-H"
+        _subdir7 = "nvme_show-regs"
```

Comment:
Likewise above, would it be possible to name variables more understandably? (i.e. `list_ctrl_dir`)?